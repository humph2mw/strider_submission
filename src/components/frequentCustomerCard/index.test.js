import React from 'react'
import TestRenderer from 'react-test-renderer'
import FrequentCustomerCard from './index'
import data from '../../pages/dashboard/receipts.json'

it('renders correctly', () => {
  const tree = TestRenderer
    .create(<FrequentCustomerCard data={data} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
