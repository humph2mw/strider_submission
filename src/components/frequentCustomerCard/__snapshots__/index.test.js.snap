// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
  id="frequentCustomerCard"
>
  <div
    className="card-title h5"
  >
    Recurring Customers
  </div>
  <table
    className="table table-striped table-hover"
  >
    <thead>
      <tr>
        <th>
          Customer Name
        </th>
        <th>
          Number of Orders
        </th>
        <th>
          Total Spent
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          Elizabeth
        </td>
        <td />
        <td>
          $undefined
        </td>
      </tr>
      <tr>
        <td>
          Emira
        </td>
        <td />
        <td>
          $undefined
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
